code/datums/components/harvester.dm

```diff
@@ -225,26 +216,23 @@
 	if(target.status_flags & INCORPOREAL || user.status_flags & INCORPOREAL) //Incorporeal beings cannot attack or be attacked
 		return
 
+	if(isxeno(target))
+		target.apply_damage(weapon.force*0.6, BRUTE, user.zone_selected)
 	switch(loaded_reagent)
 		if(/datum/reagent/medicine/tramadol)
-			target.apply_damage(weapon.force*0.6, BRUTE, user.zone_selected)
 			target.apply_status_effect(/datum/status_effect/incapacitating/harvester_slowdown, 1 SECONDS)
 
 		if(/datum/reagent/medicine/kelotane)
 			target.adjust_sunder(7.5) //Same amount as a shotgun slug
 			target.flamer_fire_act(10)
-			target.apply_damage(max(0, 20 - 20*target.hard_armor.getRating("fire")), BURN, user.zone_selected, FIRE)
-			var/list/cone_turfs = generate_cone(target, 1, 0, 181, Get_Angle(user, target.loc))
-			for(var/turf/checked_turf AS in cone_turfs)
-				for(var/mob/living/victim in checked_turf)
-					victim.flamer_fire_act(10)
-					victim.apply_damage(max(0, 20 - 20*victim.hard_armor.getRating("fire")), BURN, user.zone_selected, FIRE)
 
 		if(/datum/reagent/medicine/bicaridine)
-			if(isxeno(target))
-				return
-			INVOKE_ASYNC(src, PROC_REF(attack_async), source, target, user, weapon)
-			. = COMPONENT_ITEM_NO_ATTACK
+			INVOKE_ASYNC(src, PROC_REF(attack_bicaridine), source, target, user, weapon)
+			if(!(isxeno(target)))
+				. = COMPONENT_ITEM_NO_ATTACK
+
+		if(/datum/reagent/medicine/tricordrazine)
+			target.apply_status_effect(/datum/status_effect/shatter, 3 SECONDS)
```

Comment:
shatter is quite good, but seems kinda wack vs just using kelo for sunder and fire.

Reply:
Do you mean this is weak when compared to kelo?
Or that kelo is too strong compared to this.
I don't really know how strong of an effect Shatter is, but anytime I've gotten shattered as a xeno I always took that as a sign to GTFO, so I assumed the effect was quite powerful enough to not need any additional effect to accompany it.